packages/extension-ui/src/Popup/AuthManagement/AccountManagement.tsx

```diff
@@ -159,14 +169,16 @@ export default withRouter(styled(AccountManagement)`
     margin: 0px;
     
     ${Checkbox} {
-      margin-right: 16px;
+      margin-right: 32px;
     }
 
     .accountList {
       height: 100%;
 
+      padding-right: 8px;
+
       ${AccountsTree} {
-        width: calc(100% - 16px);
+        width: calc(100% - 32px);
```

Comment:
I think it would be best to get rid of as many of those 100% +/- `x`px as possible. Not sure if it's worth to extract a const now :thinking:

Reply:
How exactly are you thinking about getting rid of it?

Reply:
It depends where, but here it would be nice to make `AccountsTree` width an effect of page layout, not force it with a prop.

In multiple places we can use `box-sizing: border-box` instead of manually calculating the width (perhaps that could be a global default, then again is it worth the trouble?).

One more reason - SplashHandler / ScrollWrapper forcing too much padding on children, yet it still doesn't fix content shift because of scrollbar

Reply:
Good points 👍

Reply:
Not suire soon we'll be able to approach tho, seems time consuming 😅 But do as you see fit.